### PR TITLE
Feature/179 time format

### PR DESCRIPTION
**Pull request by bedletskyi**

### CODE REVIEW

File: frontend/src/helpers/time.helper.ts [hunk 1]
```diff
@@ -21,11 +21,14 @@ export const timeFormat = (seconds: number) => {
...
     const minutes = Math.floor(bufer / SECONDS_IN_MINUTE);
-    result += minutes.toString() + 'm ';
+    result += minutes.toString() + 'm';
     bufer = bufer - SECONDS_IN_HOUR * minutes;
   }
 
-  if (!result) result = '0m';
+  if (!result) {
+    if (bufer > 0) result = bufer + 's';
+    else result = '0m';
```
Comment (MaximGoncharukSW) [inline frontend/src/helpers/time.helper.ts]: Use ternary operator here. And do not use if/else wihout braces.

File: frontend/src/screens/AddCourse/containers/AddCoursePage/sagas.ts [hunk 2]
```diff
@@ -23,6 +24,7 @@ function* getCourse(action: Routine<any>) {
     const response = yield call(courseService.getCourseById, action.payload);
     yield put(fetchEditCourseRoutine.success(response));
   } catch (error) {
+    history.push('/');
```
Comment (MaximGoncharukSW) [inline frontend/src/screens/AddCourse/containers/AddCoursePage/sagas.ts]: Use the right way to call the method:
`yield call(history.push, '/')`

File: frontend/src/screens/AddPath/containers/AddPathPage/sagas.ts [hunk 3]
```diff
@@ -39,6 +40,7 @@ function* loadEditPath(action: Routine<any>) {
     const response = yield call(addPageService.getPath, action.payload);
     yield put(fetchPathToEditRoutine.success(response));
   } catch (error) {
+    history.push('/');
```
Comment (MaximGoncharukSW) [inline frontend/src/screens/AddPath/containers/AddPathPage/sagas.ts]: See the comment above.

File: frontend/src/helpers/time.helper.ts (hunk 1 above)
Comment (AndersenGans) [reply frontend/src/helpers/time.helper.ts]: result = condition ? true : false;

**Review by MaximGoncharukSW — APPROVED** [review]

**Pull request merged**